### ISSUE TITLE
No Mac Support

### ISSUE THREAD

**Issue by LuckyLion1378** [ange-yaghi/engine-sim#242]
I don't know exactly how difficult this would be, but I'm sure there's people, like myself included, dying to play this but can't due to the fact they're on Mac. please, if it is easy, try adding Mac support? once again, I don't know how this stuff works at all, just a supportive player hoping that this is easy. also I don't know if this is an "issue" but I don't know where else to put it. this is my first time on GitHub.

**Comment by BalaRohanKotni** [comment]
Duplicate of https://github.com/ange-yaghi/engine-sim/issues/221

**Comment by a1cd** [comment]
@BalaRohanKotni I think macOS has a different ways of dealing with software installation so i would argue it being a dupe of #2 . It requires different libraries for view windows (the things that you see the sim with, not the OS Windows) and for many other processes. Although it is based off of Debian kernel under the hood it is quite different from any Linux system in many ways.

**Welcome to GitHub @LuckyLion1378 !** Yes, this is an issue btw (issues can be pretty much anything from bugs to suggestions to anything (depends on repository guidelines if there are any)). 
The difficulty of macOS support can range from pretty easy but requires research to redo the entire codebase, its gonna take a miracle for it to work.

**Comment by Simsalaba** [comment]
@a1cd You mean a Darwin kernel? (Not Debian)

**Comment by UnBeatWaterGH** [comment]
> @a1cd You mean a Darwin kernel? (Not Debian)

Yup MacOS uses the Darwin kernel, plus the "Debian Kernel" isn't even a thing. I'm guessing you were reffering to the Linux kernel.

**Comment by array-in-a-matrix** [comment]
This issue is not a dupe of #2. Macos uses a completely different kernel called [XNU](https://en.wikipedia.org/wiki/XNU). Debian along with all other Linux distributions (operating systems that use Linux) use the [Linux kernel](https://en.wikipedia.org/wiki/Linux) or some modified version of it. Darwin and Debian are both operating systems that *use* the XNU and Linux kernels respectively.
